rsconnect/admin_actions.py

```diff
@@ -0,0 +1,338 @@
+"""
+Public API for administering content.
+"""
+import contextlib
+import json
+import time
+import sys
+
+# This probably breaks python2, can we remove python2.7 support from setup and/or can we require >3 for only the admin tool?
```

Comment:
This is a good question for @kellobri. Most python tooling is requiring 3.6 or later at this point. Publishing a new version that requires python3 leaves the old version available to python2 users (but strands them on that version).